Library/Formula/pgroonga.rb

```diff
@@ -0,0 +1,16 @@
+class Pgroonga < Formula
+  desc "An extension for PostgreSQL to use Groonga as an index."
+  homepage "https://pgroonga.github.io/"
+  url "http://packages.groonga.org/source/pgroonga/pgroonga-0.6.0.tar.gz"
+  sha256 "29a631f68013bba19321beac0d7b933eafce60aec1597aa8fff0b0898ff2a73d"
+
+  depends_on "cmake" => :build
+  depends_on "pkgconfig" => :build
+  depends_on "postgresql"
+  depends_on "groonga"
+
+  def install
+    system "make"
+    system "make", "install"
```

Comment:
`PGroonga` intends to install `${prefix}/Cellar/postgresql/9.4.4{/lib/pgroonga.so, /share/postgresql/extension/pgroonga--<version>.sql}`.
PGroonga's makefile dose not provide choice installation directory.
"No choice installation directory" is an big issue to Homebrew Formula?
If so, I will discuss about it at upstream.